pipeline-runner/R/gem2s-7-upload_to_aws.R

```diff
@@ -127,8 +127,8 @@ meta_sets <- function(input, scdata, color_pool) {
   # names of metadata tracks
   keys <- names(meta)
 
-  # corresponding names stored in seurat object meta.data
-  seurat_keys <- scdata@misc$metadata_lookups[keys]
+  # same values used in construct_metadata
+  seurat_keys <- make.names(c('samples', keys), unique = TRUE)[-1]
```

Comment:
I don't quite get why samples is here, because we already generate the samples cell_sets, right?

Reply:
On line 74 - 81

Reply:
it's just so that make.names has the same supplied values as it did in `create_seurat`